unit-tests/DDXParserTest.cc

```diff
@@ -82,6 +82,8 @@ class DDXParserTest: public TestFixture {
 
     CPPUNIT_TEST_SUITE (DDXParserTest);
 
+// Temporarily turn off these tests because of memory leaks. KY 2022-11-03
+#if 0
```

Comment:
Is there a ticket to pick this up in the future?

Reply:
Actually this is not ready to merge. The NcML tests failed as I briefed in the stand-up. I am checking both libdap4  and BES NcML code now. The del_attr_table()  still has some issues.